onnx/defs/math/defs.cc

```diff
@@ -2367,38 +2373,44 @@ ONNX_OPERATOR_SET_SCHEMA(
             "it is treated as if having all ones.",
             "T",
             OpSchema::Optional)
+        .Input(
+            3,
+            "ignore_index",
+            "Specifies a target value that is ignored and does not contribute to the input gradient.",
+            "Tind",
+            OpSchema::Optional)
         .Output(
             0,
             "output",
             "Weighted loss float Tensor. If reduction is 'none', this has the "
             "shape of [batch_size], or [batch_size, D1, D2, ..., Dk] in case of "
             "K-dimensional loss. Otherwise, it is a scalar.",
             "T")
-	.Output(
-	    1,
-	    "log_prob",
-	    "Log probability tensor. If the output of softmax is prob, its value is log(prob).",
-	    "T",
-	    OpSchema::Optional)
-        .TypeConstraint(
+        .Output(
```

Comment:
why is this is required now?
PyTorch only has 1 output for NllLoss/CrossEntropy

Reply:
it's optional output for softmaxcrossentropy, it was a part of the original PR

Reply:
@KsenijaS initially it was optional, but this PR makes it required.
I am curious of the reason since PyTorch does not have it as an output

Reply:
my bad, it is still optional; the "OpSchema::Optional" is after the below red lines.